packages/flutter_tools/lib/src/build_runner/resident_web_runner.dart

```diff
@@ -515,7 +515,7 @@ class _ExperimentalResidentWebRunner extends ResidentWebRunner {
           'import "$generatedImport";',
         'Future<void> main() async {',
         if (hasWebPlugins)
-          '  registerPlugins(webPluginRegistry);'
+          '  registerPlugins(webPluginRegistry);',
```

Comment:
I thought we had a lint for this

Reply:
/cc @munificent - this looks like a language gotcha

Reply:
The formatter would have likely indented the following strings and make this more obvious :man_shrugging: